HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
```

Comment:
I like naming the db model class ends with "DbModel" or "DatabaseModel" or something like that.